edgedb/protocol/codecs/object.pyx

```diff
@@ -85,7 +85,9 @@ cdef class ObjectCodec(BaseNamedRecordCodec):
         extra_args = passed_args - required_args
 
         error_message = f'expected {required_args} arguments'
-        error_message += f', got {passed_args}'
+
+        passed_args_repr = repr(passed_args) if passed_args else 'nothing'
```

Comment:
I'm not quite sure about that one, maybe use `{}` instead of `nothing`?

Reply:
`nothing` sounds okay to me